tests/units/test_units.py

```diff
@@ -5,6 +5,7 @@
 
 import sys
 
+from distutils.version import LooseVersion
```

Comment:
`distutils` is standard library, so should be above sys.

Reply:
Whoops, forgot to flake8 check on this one!